README.md

```diff
@@ -11,10 +11,30 @@
 
 🐺 φεγγάρι - The Lua VM written in JS ES6 for Node and the browser
 
+## Getting started in the browser
+
+Fengari is not released yet but you can experiment with it in the browser by following the instructions of the [fengari-web](https://github.com/fengari-lua/fengari-web) repository.
```

Comment:
I think we need a brief intro for all the other fengari repositories.
e.g. 

This repository contains the core fengari code (which is a port of the Lua C library) which includes parser, virtual machine and base libraries.
However it is rare to use the code in this repository directly.

  - To use fengari in a web browser as easily as you might use JavaScript, see [fengari-web](https://github.com/fengari-lua/fengari-web)
  - For a clone of the `lua` command line tool, but running under node.js, see [fengari-node-cli](https://github.com/fengari-lua/fengari-node-cli)
  - [fengari-interop](https://github.com/fengari-lua/fengari-interop) is a lua library that makes interoperating with JavaScript objects simple, it is already included in fengari-web.

Reply:
See force pushed changes